client/src/components/Questions/SearchBar.jsx

```diff
@@ -8,15 +8,14 @@ export default function SearchBar({
   return (
     <div className="qa search">
       <input
-        className="qa search-input"
-        name="search"
-        type="text"
+        className="qa"
+        id="question-search"
+        aria-label="Search Question"
```

Comment:
Would like to learn more about aria-label...should we be applying this to all inputs?